### PR TITLE
[hot-fix] sql getting translated

### LINKED ISSUE #11194: Spanish Translation.
There is some issue when spanish translation is selected, in the Health care Domain. 

Algo salió mal
Error de servidor: Se recomienda consultar los registros o contacte el soporte técnico.
 I think the SQL query has been translated too.  In the frappe.log file I got this.

ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '# **_seleccione * de tabPatient donde name = &#39;PID-00001&#39'_** at line 1")

### PR DESCRIPTION
**Pull request by rkurungadam**
fix #11194

### CODE REVIEW

File: erpnext/healthcare/doctype/patient/patient.py [hunk 1]
```diff
@@ -111,10 +111,10 @@ def make_invoice(patient, company):
 
 @frappe.whitelist()
 def get_patient_detail(patient, company=None):
-	patient_dict = frappe.db.sql(_("""select * from tabPatient where name='{0}'""").format(patient), as_dict=1)
+	patient_dict = frappe.db.sql("""select * from tabPatient where name=%s""", (patient), as_dict=1)
 	if not patient_dict:
 		frappe.throw("Patient not found")
-	vital_sign = frappe.db.sql(_("""select * from `tabVital Signs` where patient='{0}' order by signs_date desc limit 1""").format(patient), as_dict=1)
+	vital_sign = frappe.db.sql("""select * from `tabVital Signs` where patient=%s order by signs_date desc limit 1""", (patient), as_dict=1)
```
Comment (tundebabzy) [inline erpnext/healthcare/doctype/patient/patient.py]: `(patient)` should be `(patient,)`. The `values` parameter of `db.sql` takes a list (or tuple) or a dict.

File: erpnext/healthcare/doctype/lab_test/lab_test.py [hunk 2]
```diff
@@ -291,5 +291,5 @@ def create_invoice(company, patient, lab_tests, prescriptions):
 
 @frappe.whitelist()
 def get_lab_test_prescribed(patient):
-	return frappe.db.sql(_("""select cp.name, cp.test_code, cp.parent, cp.invoice, ct.physician, ct.consultation_date from tabConsultation ct,
-	`tabLab Prescription` cp where ct.patient='{0}' and cp.parent=ct.name and cp.test_created=0""").format(patient))
+	return frappe.db.sql("""select cp.name, cp.test_code, cp.parent, cp.invoice, ct.physician, ct.consultation_date from tabConsultation ct,
+	`tabLab Prescription` cp where ct.patient=%s and cp.parent=ct.name and cp.test_created=0""", (patient))
```
Comment (tundebabzy) [inline erpnext/healthcare/doctype/lab_test/lab_test.py]: `(patient)` should be `(patient,)`

**Review by tundebabzy — CHANGES_REQUESTED** [review]

File: erpnext/healthcare/doctype/patient/patient.py (hunk 1 above)
Comment (nabinhait) [reply erpnext/healthcare/doctype/patient/patient.py]: @tundebabzy `(patient)` is also ok.

**Review by nabinhait — APPROVED** [review]

**Comment by josmeldiaz21** [comment]
Can some one close this thread? https://discuss.erpnext.com/t/sql-queries-in-csv-spanish-translation/29417

**Pull request merged**